crypto/ed25519/private_test.go

```diff
@@ -146,3 +146,49 @@ func TestPrivateKey_Kind(t *testing.T) {
 		})
 	}
 }
+
+func TestPrivateKey_Sign(t *testing.T) {
+	tests := []struct {
+		name    string
+		pk      PrivateKey
+		msg     []byte
+		want    []byte
+		wantErr bool
+	}{
+		{
+			"success-charlotte",
+			charlottePrivateKey,
+			[]byte("message"),
+			[]byte("}Q\xea\xfaRx1i\xd0\xa9J\x0c\x9f+\xca\xd5\xe0=)\x173\x00\x93\x0f\xf3\xc7\xd6;\xfdd\x17\xae\x1b\xc8\x1f\xefQ\xba\x14\x9a\xe8\xa1\xe1\xda\xe0_\xdc\xa5\x07\x8b\x14\xba\xc4\xcf&\xcc\xc6\x01\x1e^\xabw\x03\x0c"),
```

Comment:
These byte values are really difficult to read. Not that bytes are easy to read 😕 Look at the `assert` package and the comment below. If you let the test fail you will get a "nicer" value to put here in the for of `[]byte{0xaa, 0xaa, .... }`